src/app/signup/page.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import SignupForm from '@/app/signup/components/SignupForm';
+import { Box, useTheme } from '@mui/material';
+import Image from 'next/image';
```

Comment:
This feature needs nextjs server. We need to chat with Devops team to see if we can use this or not.